scripts/delayed.js

```diff
@@ -5,3 +5,28 @@ import { loadScript, sampleRUM } from './aem.js';
 sampleRUM('cwv');
 
 loadScript('https://icici-securities.allincall.in/files/deploy/embed_chatbot_11.js?version=1.1');
+loadScript('/scripts/cookie-script.js');
+loadScript('https://assets.adobedtm.com/64c36731dbac/390f7bab5b74/launch-285ee83071cc-development.min.js', { async: true });
+
+(function gtm(w, d, s, l, i) {
+  w[l] = w[l] || [];
+  w[l].push({
+    'gtm.start': new Date().getTime(),
+    event: 'gtm.js',
+  });
+  const f = d.getElementsByTagName(s)[0];
+  const j = d.createElement(s);
+  const dl = l !== 'dataLayer' ? `&l=${l}` : '';
+  j.async = true;
+  j.src = `https://www.googletagmanager.com/gtm.js?id=${i}${dl}`;
+  f.parentNode.insertBefore(j, f);
+}(window, document, 'script', 'dataLayer', 'GTM-WF9LTLZ'));
+
+(function loadTagManagerElement() {
+  const htmlContent = '<iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WF9LTLZ"\n'
+      + 'height="0" width="0" style="display:none;visibility:hidden"></iframe>';
+  const scriptElement = document.createElement('noscript');
```

Comment:
We don't need the `noscript` as in that case this script itself will not run :)

Reply:
@sdmcraft  They have the same way in their current website as well and this is what that has been shared too. Also as per [0], `noscript` tag is supposed to work when browser does not support script or scripts has been disabled at browser. Ami I missing something here or got the interpretation of the `noscript` tag wrong ? 

[0]https://developer.mozilla.org/en-US/docs/Web/HTML/Element/noscript

Reply:
if the browser does not support javascript, then `delayed.js` where you are proposing this change would not run.

Reply:
thats true, missed this